controls/mysql_conf.rb

```diff
@@ -18,12 +19,15 @@
 
 mysql_hardening_file = '/etc/mysql/conf.d/hardening.cnf'
 
-user = attribute('User', description: 'MySQL database user', value: 'root', required: true)
-pass = attribute('Password', description: 'MySQL database password', value: 'iloverandompasswordsbutthiswilldo', required: true)
+user = input('user')
```

Comment:
Isn't 'input' deprecated in favor of 'attribute' ?

Reply:
It's the other way around: https://docs.chef.io/inspec/inputs/#which-profiles-support-inputs

> The best way for a profile to indicate it supports inputs is to list them in the metadata file, inspec.yml. Any profile that has an inputs (or the deprecated attributes) section in its inspec.yml

See also: https://github.com/inspec/inspec/pull/4008

Reply:
Ok. Weird from my part to have mixed up that.